src/semantics/ConstructorValidator.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 Traceur Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {SUPER_EXPRESSION} from '../syntax/trees/ParseTreeType.js';
+import {FindVisitor} from '../codegeneration/FindVisitor.js';
+
+class ConstructorValidator extends FindVisitor {
+  /**
+   * @param {ErrorReporter} reporter
+   */
+  constructor(reporter) {
+    super();
```

Comment:
If we pass 'false' explicitly and `// Stop seeking when found` then we don't have to read `FindVisitor` to remember how it works.